scripts/packages/ui/src/components/notifications/ToastContainer.tsx

```
@@ -39,10 +39,18 @@ export const ToastContainer = memo(function ToastContainer(): JSX.Element {
   );
 
   return (
-    <section aria-label="Notifications">
-      {visibleNotifications.map((notification) => (
-        <Toast key={notification.id} notification={notification} onDismiss={handleDismiss} />
-      ))}
-    </section>
+    <div
+      className="fixed z-[9999] pointer-events-none top-5 left-1/2 -translate-x-1/2 md:left-auto md:right-5 md:translate-x-0"
+      role="region"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)